docs/codeql/ql-language-reference/expressions.rst

```diff
@@ -308,7 +308,10 @@ The following aggregates are available in QL:
 
       rank[4](int i | i = [5 .. 15] | i)
 
-  Note that the rank indices start at ``1``, so ``rank[0](...)`` returns no results.
+  .. pull-quote:: Note
+
+     - Rank indices start at ``1``, so ``rank[0](...)`` returns no results.
+     - ``rank[1](...)`` is the same as ``min(...)``.
```

Comment:
This is a bit unrelated, but anyway: I remember reading in https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md that QLDoc shouldn't use the word `return`. Should we maybe replace "returns no result" with "has no results"?

Reply:
Oh nice, thanks! I've updated it here. Other areas of docs probably use this incorrectly too, but checking those can be a separate job 😅